src/period/index.js

```diff
@@ -8,9 +8,10 @@
 const submitPeriod = require('../txHelpers/submitPeriod');
 const { logPeriod } = require('../utils/debug');
 
-module.exports = (bridgeState, nodeConfig = {}) => async (rsp, chainInfo) => {
-  const height = chainInfo.height - 32;
-  if (height === 0) {
+module.exports = (bridgeState, nodeConfig = {}) => async (rsp, chainInfo, chainHeight) => {
+  const height = chainHeight - 32;
```

Comment:
the chainHeight in chainInfo actually conforms to the bridgeState.currentState now.
The problem is that in rare circumstances tendermint throws some non fatal errors in signAndVote and calls the `checkBridge` with blockHeight of 1 et al. 

That means `chainHeight` is the 'real' height requested in `abci.checkBridge` for whatever stupid reason it may come to such an error condition.

I did not seen such errors at all through later testing, so it may be obsolete but still nice to keep (just in case)